src/components/box/box.component.tsx

```diff
@@ -52,87 +59,70 @@ export interface BoxProps
   rowGap?: Gap;
   /** Design Token for Box Shadow. Note: please check that the box shadow design token you are using is compatible with the Box component. */
   boxShadow?: BoxShadowsType;
+  /** Set the className attribute of the Box component */
+  className?: string;
```

Comment:
```suggestion
  /** @private @ignore */
  className?: string;
```
Similar to `Typography`

Reply:
Makes sense